docs/docs/concepts/template_applications.md

```diff
@@ -0,0 +1,17 @@
+# Template Applications
+
+!!! note Prerequisites
+
+    - [LangGraph Studio](./langgraph_studio.md)
+
+Templates are open source reference applications designed to help you get started quickly when building with LangGraph. They provide working examples of common agentic workflows that can be customized to your needs.
+
+Templates can be accessed via [LangGraph Studio](langgraph_studio.md), or cloned directly from Github. You can download LangGraph Studio and see available templates [here](https://studio.langchain.com/).
+
+## Available templates
+
+- **New LangGraph Project**: A simple, minimal chatbot with memory ([Python](https://github.com/langchain-ai/new-langgraph-project), [JS/TS](https://github.com/langchain-ai/new-langgraphjs-project)).
```

Comment:
Nice -- a tab might look a bit better here 

=== Python


=== js


but also not high value atm!